sources/web/datalab/templates/edit.html

```diff
@@ -11,6 +11,8 @@
   <link rel="stylesheet" href="/static/style/style.min.css" type="text/css" />
   <link rel="stylesheet" href="/static/style/datalab.css" type="text/css" />
   <link rel="stylesheet" id="themeStylesheet" href="/static/style/custom.css" type="text/css" />
+
+  <script src="/static/components/jquery/jquery.min.js"></script>
```

Comment:
We need to add it explicitly because we are now using it before we load any Jupyter code.